overflow/github.md

```diff
@@ -0,0 +1,95 @@
+Title | Tags
+--- | ---
+Intro to GitHub | github, git
+
+## Objectives
+* Learn GitHub basics
+* Launch a GitHub pages project
+
+## TODO: Structure
+* *This is just an example structure, feel free to change*
+* 10:00 - 10:40 - Challenge problem
+* 10:40 - 10:45 Break
+* 10:45 - 11:30 Lecture part 1
+* 11:30 - 1:00 Exercise 1
+* 1:00 - 2:00 Lunch
+* 2:00 - 2:55 Lecture part 2
+* 2:55 - 3:00 Break
+* 3:00 - 4:55 Exercise 2
+* 4:55 - 5:00  Break
+* 5:00 - 6:00 - Review 
```

Comment:
Yeah still thinking on the best way to handle structure but think it'll
more or less be the structure from Rina's docs

On Tue, Jun 28, 2016 at 5:57 PM, Benjamin Sussman <notifications@github.com>
wrote:

> In overflow/github.md
> <https://github.com/C4Q/ac-curriculum-web/pull/14#discussion_r68851796>:
>
> > +## Objectives
> > +* Learn GitHub basics
> > +* Launch a GitHub pages project
> > +
> > +## TODO: Structure
> > +* *This is just an example structure, feel free to change*
> > +* 10:00 - 10:40 - Challenge problem
> > +* 10:40 - 10:45 Break
> > +* 10:45 - 11:30 Lecture part 1
> > +* 11:30 - 1:00 Exercise 1
> > +* 1:00 - 2:00 Lunch
> > +* 2:00 - 2:55 Lecture part 2
> > +* 2:55 - 3:00 Break
> > +* 3:00 - 4:55 Exercise 2
> > +* 4:55 - 5:00  Break
> > +* 5:00 - 6:00 - Review
>
> SHould we take the structure from Rina's docs on time breakdown? And not
> have a custom structure for each class?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/C4Q/ac-curriculum-web/pull/14/files/ba8677ef4df6f0410feb371b264fff35e63cb86c#r68851796>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAQnnv5n9DrDdGmy6Cn73FUIighOElbWks5qQZjcgaJpZM4JAYSc>
> .
>